src/metrics/prometheus/task.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from functools import wraps
+from time import perf_counter
+from typing import Callable
+
+from src.metrics.prometheus.basic import TASKS_DURATION, Status
+
+logger = logging.getLogger(__name__)
+
+
+def task(name: str):
```

Comment:
duration_meter maybe?
Task is something that you run separately from main process or in another thread

Reply:
fixed